.github/workflows/on_pull_request_package_size.yml

```diff
@@ -1,9 +1,12 @@
-name: Package Size
+name: Package size
 
-on: pull_request
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review]
```

Comment:
Maybe we can remove `ready_for_review` here since nothing should change in the code between draft -> rdy that isn't already caught

Reply:
I think I'm going to just scope it to on: `pull_request` and not care about types.